reco_utils/dataset/pandas_df_utils.py

```diff
@@ -73,7 +73,7 @@ def filter_by(df, filter_by_df, filter_by_cols):
     ]
 
 
-def libffm_converter(df, col_rating=DEFAULT_RATING_COL, filepath=None):
+class LibffmConverter():
```

Comment:
if we are going to use a class (see comment above), it is a good practice to inhereit from object: `LibffmConverter(object)`

Reply:
It should be equivalent in this case see [here](https://stackoverflow.com/questions/4015417/python-class-inherits-object).

Reply:
cool